indico/web/forms/fields.py

```diff
@@ -1021,12 +1021,30 @@ class IndicoTagListField(HiddenFieldList):
 
 
 class IndicoMarkdownField(TextAreaField):
-    def __init__(self, *args, **kwargs):
-        kwargs.setdefault('description', _("You can use Markdown or basic HTML formatting tags."))
-        super(IndicoMarkdownField, self).__init__(*args, **kwargs)
+    """A Markdown-enhanced textarea.
+
+    When using WMD you need to include the markdown JS/CSS bundles
+    and also the MathJax JS bundle (even when using only WMD without
+    Mathjax).
+
+    :param wmd: Whether to use the WMD widget with its live preview
+    :param mathjax: Whether to use MathJax in the WMD live preview
+    """
 
     widget = JinjaWidget('forms/markdown_widget.html', single_kwargs=True, rows=5)
 
+    def __init__(self, *args, **kwargs):
+        self.use_wmd = kwargs.pop('wmd', False)
```

Comment:
Do we really want to call it `wmd`? I'd use something more general like `editor`.